src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.0.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>1.1.0</RoslynSemanticVersion>
```

Comment:
I don't see anything that would require changing the version in here. [Why] is this necessary?